docs/the_black_code_style.md

```diff
@@ -289,6 +289,16 @@ If you are adopting _Black_ in a large project with pre-existing string conventi
 you can pass `--skip-string-normalization` on the command line. This is meant as an
 adoption helper, avoid using this for new projects.
 
+As an experimental option, _Black_ splits long strings (using parentheses where
+appropriate) and merges short ones. When split, parts of f-strings that don't need
+formatting are converted to plain strings. User-made splits are respected when they do
+not exceed the line length limit. Line continuation backslashes are converted into
+parenthesized strings. Unnecessary parentheses are stripped. To enable experimental
+string processing, pass `--experimental-string-processing` on the command line.
+
+_Black_ indents docstrings to the appropriate level, but the actual contents are not
+processed further.
```

Comment:
Even without taking into account any of the many docstring PRs currently proposed, this is false. For multi-line docstrings, starting and ending whitespace is stripped (one of PRs is fixing Black to also do this for single-line docstrings). 

```console
ichard26@acer-ubuntu:~/programming/oss/black$ cat test.py --show-ends 
def test8():$
    """  $
    A$
    Or like this.  $
    """$
ichard26@acer-ubuntu:~/programming/oss/black$ black test.py 
reformatted test.py
All done! ✨ 🍰 ✨
1 file reformatted.
ichard26@acer-ubuntu:~/programming/oss/black$ cat test.py --show-ends 
def test8():$
    """$
    A$
    Or like this.$
    """$
```

Reply:
I think I'll have a look at the tests as Jelle proposed, add in information from them (since that is *certainly* expected behavior), and leave the rest for other issues to deal with.